experiments/functional/cspc-pool/block-deletion-with-volume/README.md

```diff
@@ -0,0 +1,16 @@
+## Test Case to validate the deletion of cspc pool is blocked if the pool has volumes.
+
+## Description
+   - This test case is capable of validate the pool deletion is blocked if the volumes are present in the pool
```

Comment:
This test case is capable of validating if the pool deletion is blocked when it has volumes.